documentation/API_documentation/QoSProfile_Mapping_Table.md

```diff
@@ -1,11 +1,25 @@
+| QOS\_M   |DL upto 30Mbps. The 5G System throughput is prioritized up to 30Mbps. At high load, the throughput is capped at 30Mbps and can be degraded to lower effective throughput (without a minimum level). | The service is only provided, when the UE is located within the home network. |
+| QOS\_S   | DL upto 10Mbps. The 5G System throughput is dprioritized up to 10Mbps. At high load, the throughput is capped at 10Mbps and can be degraded to lower effective throughput (without a minimum level). | The service is only provided, when the UE is located within the home network. |
+
+**Note:**
+The QOS profile labels in the table can be used within Camara for validating the QoD APIs.
+
+Example 2: Considering confined area network usage.
+| QOS Profile labels | Network Service Description | Network Service Constraints |
 |----------| --------- | ----------- |
-| QOS\_E   | Latency stays stable under congestion (throughput upto 2Mbps) | 7 |
-| QOS\_L   | DL upto 100Mbps (unlimited?) | 6 |
-| QOS\_M   | DL upto 30Mbps | 6 |
-| QOS\_S   | DL upto 10Mbps | 6 |
+| QOS\_E   | Latency stays stable under congestion (throughput upto 2Mbps). The application bitrate should not exceed 2Mbps. | The service is only provided, when the UE is located within the geographical area, as agreed in the Service Level Agreement.<br>The access to the network service is restricted to a maximal number of simultaneous session. |
+| QOS\_L   | The 5G System throughput is assured minimum bitrate of 20Mbps and up to 40Mbps. Higher bitrates are possible, but not assured | The service is only provided, when the UE is located in the geographical area as agreed as part of the Service Level Agreement. When the UE is located outside of the geographical area, the 5G System rejects the requested QOS profile.<br>The access to the network service is restricted to a maximal number of simultaneous session. |
+| QOS\_M   | The 5G System throughput is assured for at least 5Mbps and up to 15Mbps. Higher bitrates are possible, but not assured | The service is only provided, when the UE is located in the geographical area as agreed as part of the Service Level Agreement. When the UE is located outside of the geographical area, the 5G System rejects the requested QOS profile.<br>The access to the network service is restricted to a maximal number of simultaneous session. |
+| QOS\_S   | The 5G System throughput is assured for at least 500kbps and up to 1Mbps. Higher bitrates are possible, but not assured. | The service is only provided, when the UE is located in the geographical area as agreed as part of the Service Level Agreement. When the UE is located outside of the geographical area, the 5G System rejects the requested QOS profile.<br>The access to the network service is restricted to a maximal number of simultaneous session. |
```

Comment:
What do you mean by "assured" here? What is expected to happen if the minimum bitrate cannot be met in parts of the agreed geographical area because of the radio conditions in which the end user device finds itself (for example, if the end user requests a service for central London, but then gets on an underground train)? Given that mobile operators cannot break the laws of physics, there will be circumstances under which the minimum bitrate cannot be assured in any location, and if the API caller accepts that it cannot be completely assured, then it just becomes a "best effort" service.

Also, how is the agreed geographical area to be specified (this is not part of the current API definition), and how is the mobile operator expected to track the current location of the end user device for the purposes of determining if the assurance has been met or not? If this is based on a group of cells (base-stations), then the end user on the underground train will say "I was in central London and I didn't get my 20 Mb/s", and we'll say "You weren't connected to one of the agreed cells". Technically, we would be correct, but the end user would perceive that the service was not delivered as advertised.